.storybook/Dropdown.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Box, Card, Dropdown, DropdownField } from '../src'
+
+const DropdownMenu = Card.extend`
+  height: 200px;
+  overflow: scroll;
```

Comment:
When setting `overflow: scroll` it's good practice to also set `-webkit-overflow-scrolling: touch` for iOS

Reply:
Anther great CSS rule to learn. Thanks. Updated.